### PR TITLE
updated code test to consume a public graphql API

### PR DESCRIPTION
**Pull request by AlecAivazis**
This PR updates the frontend code test to be based around a publically available graphql API (Yelp's). I tried to have very similar requirements to the previous version in order to rest knowledge of react as well without overloading the candidate

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
 DataScience's Frontend Code Test
 ==================
 
-This is an at-home exercise that we use as part of our standard interview process for frontend and full-stack developers.
+This is an at-home exercise that we use as part of our standard interview process for frontend and full-stack developers. It requirements are designed to provide us with enough insight to evaluate your skillset, but feel free to expand on its feature set if you have time (not required).
```
Comment (locnguyen) [inline README.md]: Missing an `s`, should be `Its requirements are designed`

File: README.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
+
+In this test, you will be using the public GraphQL api provided by Yelp to
```
Comment (locnguyen) [inline README.md]: Can you capitalize `api`?

File: README.md [hunk 3]
```diff
@@ -13,10 +16,11 @@ This is an at-home exercise that we use as part of our standard interview proces
 
 Using the provided JSON data representing a collection of meal recipes, create a micro frontend application that meets the following criteria:
 
-* Display a list (or table) of recipes.
-* Allow filtering of recipes by a single ingredient.
-* Add checkboxes to allow selection of multiple recipes.
-* Show an alphabetically ordered list of distinct ingredients for the selected recipes. This should update as recipes are selected / unselected.
-* Persist the selections locally and regenerate the view on page refresh.
+* Use the public graphql api provided by yelp, for more information, see [here](https://www.yelp.com/developers/graphql/guides/intro).
```
Comment (locnguyen) [inline README.md]: Can you capitalize `yelp`?

**Comment by AlecAivazis** [comment]
@locnguyen what do you think about requiring a fork of the project instead of allowing the candidate to upload a zip? My thinking is that it would be useful to be able to watch their git commits as the candidate works through it

**Comment by locnguyen** [comment]
I don't know if I would personally have the bandwidth to review commits as they come in. But if anyone on the team might like to do that we can make it a requirement.

**Pull request merged**